crates/shared/src/lib.rs

```diff
@@ -3,7 +3,7 @@
 #[macro_use]
```

Comment:
It looks like this file accidentally became executable?

I'm actually really curious, I've seen this happen from time to time, do you know what might cause it?

Reply:
Yeah, I've seen it too, and I have no idea why. It is possible a mistyped emacs key chord does it? I really don't know... I'm certainly not running `chmod +x`! :-p